app/views/drivers/index.html.erb

```diff
@@ -0,0 +1,31 @@
+<h1>Drivers</h1>
+
+<div class="pagination">
+  <%= will_paginate @drivers %>
+</div>
+
+<% if @drivers.length > 0 %>
+  <table>
+    <tr>
+      <th>ID</th>
+      <th>Name</th>
+      <th>Vin</th>
+      <th>Status</th>
+    </tr>
+    <% @drivers.each do |driver| %>
+      <tr>
+        <td><%= driver.id %></td>
+        <td><%= link_to driver.name, driver_path(driver.id)%></td>
+        <td><%= driver.vin %></td>
+        <td><%= ActiveModel::Type::Boolean.new.cast(driver.available) ? "Available" : "Unavailable" %></td>
```

Comment:
This is the kind of code that would have been simpler if you had stored this in the database as a boolean.